### ISSUE TITLE
scan for availability during import from local drive

### ISSUE THREAD

**Issue by mrpau-richard** [learningequality/kolibri#4194]
### Observed behavior
When I import channel content from attached drive and I select the contents that's not at my local drive, it will start to download but after the download finished the channel local contents still the same.

### Expected behavior
Downloaded channel contents should show at the channel local contents. 

### Errors and logs
![screen shot 2018-08-27 at 3 59 24 pm](https://user-images.githubusercontent.com/8663934/44648007-519a4c80-aa12-11e8-87d3-fee2beb4fadb.png)
This is after I selected the local drive channel import.
![screen shot 2018-08-27 at 3 52 48 pm](https://user-images.githubusercontent.com/8663934/44648045-6bd42a80-aa12-11e8-8351-7a0bba511d82.png)
This is after I select import 53mb channel content.
![screen shot 2018-08-27 at 3 53 01 pm](https://user-images.githubusercontent.com/8663934/44648075-81e1eb00-aa12-11e8-8961-92c59a34ff0e.png)
The channel content still the same 43mb.

### Steps to reproduce
*  Download less than 1gb channel contents from the studio.
* Export all the channel contents to the local drive.
* Then select Import from local drive and select the channel that you recently exported.
* Select contents that is not on your device and import it. 

### Context

Tell us about your environment, including:
 * Kolibri version: 0.10.2
 * Operating system: Windows 7
 * Browser: Firefox 61.0.1

**Comment by lyw07** [comment]
Hi @mrpau-richard , could you please also attach the `kolibri.log` here? Thank you!

**Comment by mrpau-richard** [comment]
Hi @lyw07 , Here's the [kolibri.log](https://github.com/learningequality/kolibri/files/2326324/kolibri.log) file.

**Comment by lyw07** [comment]
thank you for the log file @mrpau-richard ! I see all of the files were failed to import from external drive. For example,
```
ERROR 2018-08-27 19:50:13,802 importcontent An error occured during content import: [WinError 2] The system cannot find the file specified: 'C:\\KOLIBRI_DATA\\content\\storage\\7\\d\\7d5f88ee1abe3b29fe80d8767e56a9f8.perseus'
```
Not sure what caused this issue. I will try to reproduce it on my VM

**Comment by benjaoming** [comment]
@mrpau-richard can you see any relation to #3269? The issue indicates that channel contents are somehow lifted from the online source, rather than the local drive...

**Comment by mrpau-richard** [comment]
@benjaoming yeah, I think this is related to the #3269 issue.

**Comment by lyw07** [comment]
I think both #3269 and this issue are caused by the limitation we have regarding local import. During local import from usb, we don't actually scan the KOLIBRI_DATA directory inside the usb. So in the content import page, content that is not available in the local drive will still be shown. We skip importing the content since it's not actually available in the local drive. That's why the log file contains the error log `The system cannot find the file specified`.

**Comment by benjaoming** [comment]
@lyw07 - ah okay, I can understand that we don't scan to check availability. But perhaps we should report back to the user about the outcome of the import, when certain resources aren't found?

Should we open a new issue for this?

Tagging #3494

**Comment by lyw07** [comment]
@benjaoming Yes I agree. There is no enough information currently to show the users, which makes them confused. 
I just searched in our github issues and noticed that there are around 4 open issues reporting this problem. I think it would be better to discuss this with the whole team about when and how are we planning to tackle this problem. I will bring it up during the dev meeting next week!

**Comment by indirectlylit** [comment]
I've [created a proposal](https://docs.google.com/document/d/1KBLNejVu7dlhZdAodk59cEHQQYZtO7BNe5B6Xz7gi5s/edit#) for how we can address this issue in the scope of 0.12

@lyw07 would you mind adding references in that doc to any related issues you're aware of, and let me know if you think they'll be addressed by the proposal?

**Comment by lyw07** [comment]
sure! I will work on that :)

**Comment by lyw07** [comment]
@indirectlylit I think the issues are well addressed in the doc. Thank you for preparing this proposal! I also added some information and links to it.

**Comment by indirectlylit** [comment]
addressed in ongoing 0.13 content management work
